src/templates/en/2019/base.html

```diff
@@ -56,9 +56,7 @@
       <section class="nav">
         <a class="navigation-logo" href="{{ url_for('home', year=year, lang=lang) }}"><img src="/static/images/logo.png" alt="web almanac by http archive"/></a>
         <nav class="nav-items">
-          <a class="navigation-logo" href="https://httparchive.org">
-            <img src="/static/images/ha-home.png" alt="HTTP Archive home" />
-          </a>
+          <a href="{{ url_for('table_of_contents', year=year, lang=lang) }}">Table of Contents</a>
```

Comment:
Now that the desktop and mobile footers contain the same content, could we merge this section with `footer-mobile-nav` and use media queries to style their positioning?

Reply:
Actually I'll merge this as-is and file a new issue to handle that.